pages/3_⚖️_Environmental_Justice.py

```diff
@@ -114,95 +142,35 @@ def main():
   c2 = st.container()
 
   with c2:
-    # EJ parameters we are working with
-    ej_parameters = {
-      "MINORPCT",
-      "LOWINCPCT",
-      "LESSHSPCT",
-      "LINGISOPCT",
-      "UNDER5PCT",
-      "OVER64PCT",
-      "PRE1960PCT",
-      "UNEMPPCT",
-      "VULEOPCT",
-      "DISPEO",
-      "DSLPM",
-      "CANCER",
-      "RESP",
-      "PTRAF",
-      "PWDIS",
-      "PNPL",
-      "PRMP",
-      "PTSDF",
-      "OZONE",
-      "PM25",
-      "UST"
-    }
-
-    @st.cache_data
-    def get_metadata():
-      columns = pd.read_csv("https://raw.githubusercontent.com/edgi-govdata-archiving/ECHO-SDWA/main/2021_EJSCREEEN_columns-explained.csv")
-      return columns
-    columns = get_metadata()
-    columns = columns.loc[columns["GDB Fieldname"].isin(ej_parameters)][["GDB Fieldname", "Description"]]
-    columns.set_index("Description", inplace = True)
-    ej_dict = columns.to_dict()['GDB Fieldname']
-
-    options = ej_dict.keys() # list of EJScreen variables that will be selected
```

Comment:
Wait, we still want this dictionary as well! "ejvar" is the behind-the-scenes variable, "ej_dict" is the human-readable name for the category, and "ejdefs" is the longform description